.github/workflow_templates/trivy-db-schedule.yml

```diff
@@ -34,5 +34,7 @@ jobs:
       - name: Download crane and copy image
         run: |
           curl -sSfL https://github.com/google/go-containerregistry/releases/download/v0.13.0/go-containerregistry_Linux_x86_64.tar.gz | tar -xzf - && chmod +x crane
-          ./crane copy ghcr.io/aquasecurity/trivy-db:2 ${{secrets.DECKHOUSE_REGISTRY_HOST}}/security/trivy-db:2
-          ./crane copy ghcr.io/aquasecurity/trivy-db:2 ${{secrets.DECKHOUSE_DEV_REGISTRY_HOST}}/security/trivy-db:2
+          ./crane copy ghcr.io/aquasecurity/trivy-db:2 ${{secrets.DECKHOUSE_REGISTRY_HOST}}/deckhouse/ce/security/trivy-db:2
```

Comment:
I'm not sure, but do we really need trivi-db in CE?

Reply:
Huh.

Reply:
Makes sense, thanks. Fixed.